exporter/importables.go

```diff
@@ -1480,4 +1491,52 @@ var resourcesMap map[string]importable = map[string]importable{
 			{Path: "library.whl", Resource: "databricks_dbfs_file", Match: "dbfs_path"},
 		},
 	},
+	"databricks_directory": {
+		Service: "directories",
+		Name: func(ic *importContext, d *schema.ResourceData) string {
+			name := d.Get("path").(string)
+			if name == "" {
+				return d.Id()
+			} else {
+				name = nameNormalizationRegex.ReplaceAllString(name[1:], "_") + "_" +
+					strconv.FormatInt(int64(d.Get("object_id").(int)), 10)
+			}
+			return name
+		},
+		List: func(ic *importContext) error {
+			notebooksAPI := workspace.NewNotebooksAPI(ic.Context, ic.Client)
+			directoryList, err := notebooksAPI.ListDirectories("/", true)
+			if err != nil {
+				return err
+			}
+			for offset, directory := range directoryList {
+				if strings.HasPrefix(directory.Path, "/Repos") {
+					continue
+				}
+				ic.Emit(&resource{
+					Resource: "databricks_directory",
+					ID:       directory.Path,
+				})
+				if offset%50 == 0 {
+					log.Printf("[INFO] Scanned %d of %d directories",
+						offset+1, len(directoryList))
+				}
+			}
+			return nil
+		},
```

Comment:
This is not necessary. It's just enough to emit `databricks_directory` from `databricks_notebook` import

Reply:
if we don't separate them out, users won't have the option to only export notebook or directory?

Reply:
Emitting of only directories could be strange requirement - who will need a directory hierarchy without any content?

Reply:
maybe for auditing purpose, especially when the parent directory permission ACLs are different from children notebooks'?

Reply:
it should be either covered by the difference of directory permissions from its parent, or permissions on individual notebooks.

P.S. You may see that not all resources have a `List` method, for example, `databricks_user`, or `databricks_dbfs_file` - in most cases they will be covered by emitting from other resources